regression-tests/backends/gmysql-master

```diff
@@ -23,9 +23,11 @@ gmysql-dbname=$GMYSQLDB
 gmysql-user=$GMYSQLUSER
 gmysql-host=$GMYSQLHOST
 gmysql-password=$GMYSQLPASSWD
+
+any-to-tcp=no
 __EOF__
 
-		gsql-master gmysql
+		gsql-master gmysql dyndns
```

Comment:
There are many ANY queries in the  dyndns tests.
And testing dyndns only via tcp feels wrong because you can't detect truncation issues with tcp